wrapper/src/test/java/integration/container/tests/PerformanceTest.java

```diff
@@ -163,7 +164,7 @@ private void execute_FailureDetectionTime_EnhancedMonitoringEnabled(
     PLUGINS.set(props, "efm");
 
     final PerfStatMonitoring data = new PerfStatMonitoring();
-    doMeasurePerformance(sleepDelayMillis, REPEAT_TIMES, props, false, data);
+    doMeasurePerformance(sleepDelayMillis, REPEAT_TIMES, props, data);
```

Comment:
does that mean that what was false is now true?

Reply:
No, the parameter was removed altogether from the doMeasurePerformance method and the failover mode does not default to being read onlt.